### ISSUE TITLE
[Raspberry pi 3B]

### ISSUE THREAD

**Issue by ToCenTek** [benkuper/Chataigne#144]
**Environment**
 - OS: [Linux raspberrypi 5.15.61-v7+ #1579 SMP Fri Aug 26 11:10:59 BST 2022 armv7l GNU/Linux]
 - img: [2022-09-22-raspios-bullseye-armhf.img]
 - Version [Chataigne-linux-armhf-1.9.8.AppImage]

**Describe the bug**
error when trying to run:
./Chataigne-linux-armhf-1.9.8.AppImage 
Chataigne: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory

**Comment by benkuper** [comment]
Yes, the CI was updated to Ubuntu 20.04 and the ssl lib is not the same now. If you can install libssl 3 on raspberry and provide here the libssl.so.3 file, I'll add it to the AppImage

**Comment by eLandon** [comment]
Please find enclosed the libs compiled on rpi, chataigne is still not starting though with errors regarding glib

_Chataigne: /lib/arm-linux-gnueabihf/libm.so.6: version GLIBC_2.29' not found (required by Chataigne)
Chataigne: /lib/arm-linux-gnueabihf/libm.so.6: version GLIBC_2.35' not found (required by Chataigne)
Chataigne: /lib/arm-linux-gnueabihf/libstdc++.so.6: version GLIBCXX_3.4.30' not found (required by Chataigne)
Chataigne: /lib/arm-linux-gnueabihf/libstdc++.so.6: version GLIBCXX_3.4.29' not found (required by Chataigne)
Chataigne: /lib/arm-linux-gnueabihf/libc.so.6: version GLIBC_2.34' not found (required by Chataigne)
Chataigne: /lib/arm-linux-gnueabihf/libc.so.6: version GLIBC_2.32' not found (required by Chataigne)_

[libssl.so.zip](https://github.com/benkuper/Chataigne/files/9849963/libssl.so.zip)

**Comment by benkuper** [comment]
Does this help ?
https://stackoverflow.com/questions/72513993/how-install-glibc-2-29-or-higher-in-ubuntu-18-04

**Comment by benkuper** [comment]
or this 
http://forums.purebasic.com/english/viewtopic.php?p=578153

**Comment by benkuper** [comment]
I manually compiled on rpi bullseye, it's not a definite fix but at least the latest stable is available for rpi without compiling

**Comment by eLandon** [comment]
Hi Ben and thanks for the fix.
I still had issues with some of my pi, mainly 

> Chataigne: /lib/arm-linux-gnueabihf/libpthread.so.0: version `GLIBC_2.30' not found (required by Chataigne)
Chataigne: /lib/arm-linux-gnueabihf/libm.so.6: version `GLIBC_2.29' not found (required by Chataigne)

due to my OS being older than the one you used for compilation I guess (buster vs bullseye). Long story short I'm using reterminal from seeedstudio and upgrading os might break things, so I'm trying to avoid that. 

I ended up compiling with an older OS. I need to try it on a up-to-date raspiOS to ensure it is working, but I guess it should. I could send you the resulting appimage if you wish
